File: skiCentar/skiCentar.Model/Resort.cs
namespace skiCentar.Model
{
    public class Resort
    {
        public int Id { get; set; }

        public string Name { get; set; } = null!;

        public string Location { get; set; } = null!;

        public int? Elevation { get; set; }

        public string SkiWorkHours { get; set; } = null!;

    }
}

File: skiCentar/UI/skicentar_desktop/lib/models/resort.dart

import 'package:json_annotation/json_annotation.dart';

part 'resort.g.dart';
@JsonSerializable()
class Resort{
  int? id;
  String? name;
  String? location;
  int? elevation;
  String? skiWorkHours;
  
  Resort({this.id,this.name});

  factory Resort.FromJson(Map<String,dynamic> json) => _$ResortFromJson(json);

  Map<String,dynamic> toJson() => _$ResortToJson(this);
}